src/CarbonAware.WebApi/src/Controllers/CarbonAwareController.cs

```diff
@@ -73,15 +73,18 @@ public async Task<IActionResult> GetEmissionsDataForLocationsByTime([FromQuery(N
     {
         using (var activity = Activity.StartActivity())
         {
-            IEnumerable<Location> locationEnumerable = CreateLocationsFromQueryString(locations);
+            IEnumerable<Location> locationEnumerable = CreateMultipleLocationsFromStrings(locations);
             var props = new Dictionary<string, object?>() {
-                { CarbonAwareConstants.Locations, locationEnumerable },
+                { CarbonAwareConstants.MultipleLocations, locationEnumerable },
                 { CarbonAwareConstants.Start, time },
                 { CarbonAwareConstants.End, toTime},
                 { CarbonAwareConstants.Duration, durationMinutes },
             };
 
-            return await GetEmissionsDataAsync(props);
```

Comment:
Should this be logged at debug level?

Reply:
I don't feel that strongly about it being debug vs information so can change.